server/services/version.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { ObjectId } from 'mongodb'
+import { ApprovalStates } from '../models/Deployment'
+import '../utils/mockMongo'
+import VersionModel from '../models/Version'
+import { createVersion, findModelVersions, findVersionById, findVersionByName, serializedVersionFields } from './version'
+import UserModel from '../models/User'
+
+const modelId = new ObjectId
```

Comment:
This is bad / archaic syntax.  You should use `new ObjectId()`.

Reply:
As said this was a typo and was resolved :)